staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/mockarmclient/interface.go

```diff
@@ -1,7 +1,7 @@
 // +build !providerless
 
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright The Kubernetes Authors.
```

Comment:
Why are we dropping the year?

Reply:
Hello @cheftako 
Thank You very much for reviewing the PR :)

The mockgen command adds below text to the autogenerated mock files

```
// Code generated by MockGen. DO NOT EDIT.
// Source: interface.go
```


because of this behavior, if we add YEAR in automatically generated files then the `verify-boilerplate.sh` fails, whenever it matches `DO NOT EDIT.` text in the file

https://github.com/kubernetes/kubernetes/blob/edb0a72cff0e43bab72a02cada8486d562ee1cd5/hack/boilerplate/boilerplate.py#L226-L228